src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -237,6 +240,13 @@ public int instanceMain(final String[] argv) {
         if (System.getProperty("ga4gh.client_secrets") == null) {
           System.setProperty("ga4gh.client_secrets", GA4GH_CLIENT_SECRETS);
         }
+
+        if (System.getProperty("google_project_requester_pays") == null && REQUESTER_PAYS_PROJECT != null) {
+            System.setProperty("google_project_requester_pays", REQUESTER_PAYS_PROJECT);
+            Log.getInstance(this.getClass()).info(
```

Comment:
Should this only be output if the google libraries are loaded?

Reply:
if the libraries are not loaded, what's the meaning of having "requestor pays"? is there something you'd like it to output in that case?